lighthouse-core/audits/dobetterweb/geolocation-on-start.js

```diff
@@ -40,15 +48,26 @@ class GeolocationOnStart extends Audit {
     if (typeof artifacts.GeolocationOnStart === 'undefined' ||
         artifacts.GeolocationOnStart === -1) {
       return GeolocationOnStart.generateAuditResult({
-        rawValue: false,
-        debugString: 'Unable to get geolocation values.'
+        rawValue: -1,
+        debugString: 'GeolocationOnStart gatherer did not run'
```

Comment:
I guess that's fine for here, then, but we really need to not eat any errors (see also #812)